internals/daemon/daemon.go

```diff
@@ -717,21 +717,79 @@ func (d *Daemon) doReboot(sigCh chan<- os.Signal, waitTimeout time.Duration) err
 	return fmt.Errorf("expected reboot did not happen")
 }
 
-var shutdownMsg = "reboot scheduled to update the system"
+const rebootMsg = "reboot scheduled to update the system"
```

Comment:
Since we are using this once, can we just use the string literal instead and add the new variable if/when we need it in the future?

Reply:
I am going to limit the scope of the changes to an absolute minimum. Your proposal makes sense, but this message should be properly considered in the bigger context, which I think is more suited for a separate exercise.